packages/expo-updates/e2e/__tests__/fixtures/project_files/e2e/environment.js

```diff
@@ -0,0 +1,37 @@
+const {
+  DetoxCircusEnvironment,
+  SpecReporter,
+  WorkerAssignReporter,
+} = require('detox/runners/jest-circus');
+
+class CustomDetoxEnvironment extends DetoxCircusEnvironment {
```

Comment:
FYI: detox v20 is moving away this setup: https://wix.github.io/Detox/docs/guide/migration#custom-jest-environment. we have detox v20 in bare-expo already.
i know the current status quo is because @config-plugins/detox is still on detox v19.